base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -213,7 +214,8 @@ Result<StatusFetcherOutput> StatusFetcher::FetchStatus(
     instances.emplace_back(instance_record_result->first);
   } else {
     if (status_the_group_flag) {
-      instances = instance_group.Instances();
+      instances = std::vector<cvd::Instance>(instance_group.Instances().begin(),
+                                             instance_group.Instances().end());
```

Comment:
nit: this assumes `Instances()` returns a reference to the same object between both calls. Can the returned value/reference from `Instances()` be stored in a local variable?

Reply:
Done